DCC-Tools~/Blender/unity_6way.py

```diff
@@ -848,12 +869,19 @@ class Properties(bpy.types.PropertyGroup):
                 min = 1,
             )
             tiling: bpy.props.IntVectorProperty(
-                name = "Tiling",
-                description = "Tiling",
+                name = "Rows / Columns",
+                description = "Rows and columns of the flipbook",
                 default = (1,1),
                 size = 2,
                 min = 1,
             )
+
+            frame_step: bpy.props.IntProperty(
```

Comment:
Would it make sense to move the frame step to the top options, next to the frame range?
It seems useful to be able to skip rendering frames, specially if they are later not included in the flipbook.
I think image compositor nodes also have a step setting to skip frames in a sequence

Reply:
Hey! It makes sense, however I had the following train of thought in this case:

- I've experimented with putting frame step up there and rendering frames selectively, and when i wanted to experiment with different flipbook layouts (4x4 16f vs 8x8 64f etc) it felt that rendering the entire frame range to subsequently reexport flipbooks with one click gives me more freedom
- That being said, I've used EEVEE for test renders which is very quick, but using Cycles for final shots would definitely require optimized frame skipping. I was just unsure how to organize UX for cases like "I need a full frame range, but just skip frames in Flipbook" etc. So for now opted out for a crude solution so later we can go back to this.